website/components/remote-plugin-docs/utils/fetch-plugin-docs.js

```diff
@@ -13,7 +13,10 @@ const parseDocsZip = require('./parse-docs-zip')
 // docs files were missing or invalid, with a path to resolution
 async function fetchDocsFiles({ repo, tag }) {
   // If there's a docs.zip asset, we'll prefer that
-  const docsZipUrl = `https://github.com/${repo}/releases/download/${tag}/docs.zip`
+  const docsZipUrl =
+    tag === 'latest'
+      ? `https://github.com/${repo}/releases/latest/download/docs.zip`
```

Comment:
This fixes an issue where the links to the `latest` `docs.zip` would always 404, because I didn't construct them correctly 😬

Reply:
Good catch.